From 4649f2a11fe4b87e9b79f50ff8ac99ba9a2344f4 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Michael=20Br=C3=BCning?= <michael.bruning@qt.io>
Date: Wed, 25 Oct 2017 09:47:47 +0200
Subject: [PATCH] Build accessibility tests only when Qt is configured with
 accessibility

Original patch by Daniel Mack <daniel@zonque.org>

Task-number: QTBUG-64015
Change-Id: I7dd0a1aa9bff08c6d19e818acf0e1b8b4b701f5b
Reviewed-by: Alexandru Croitor <alexandru.croitor@qt.io>
---
 tests/auto/widgets/widgets.pro | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/tests/auto/widgets/widgets.pro b/tests/auto/widgets/widgets.pro
index 441eea0..7bab18f 100644
--- a/tests/auto/widgets/widgets.pro
+++ b/tests/auto/widgets/widgets.pro
@@ -1,7 +1,6 @@
 TEMPLATE = subdirs
 
 SUBDIRS += \
-    qwebengineaccessibility \
     qwebenginedefaultsurfaceformat \
     qwebenginedownloads \
     qwebenginefaviconmanager \
@@ -14,6 +13,10 @@ SUBDIRS += \
     qwebenginesettings \
     qwebengineview
 
+qtConfig(accessibility) {
+    SUBDIRS += qwebengineaccessibility
+}
+
 contains(WEBENGINE_CONFIG, use_spellchecker):!cross_compile {
     !contains(WEBENGINE_CONFIG, use_native_spellchecker) {
         SUBDIRS += qwebenginespellcheck
-- 
2.7.4

